src/Filters/SessionAuth.php

```diff
@@ -51,10 +51,18 @@ public function before(RequestInterface $request, $arguments = null)
 
             // Block inactive users when Email Activation is enabled
             $user = $authenticator->getUser();
+
+            if ($user->isBanned()) {
+                $authenticator->logout();
+
+                return redirect()->to(config('Auth')->logoutRedirect())
+                    ->with('error', lang('Auth.logOutBannedUser'));
+            }
+
             if ($user !== null && ! $user->isActivated()) {
                 $authenticator->logout();
 
-                return redirect()->route('login')
+                return redirect()->to(config('Auth')->logoutRedirect())
```

Comment:
Why did you change? Is this related to PR?

Reply:
No it isn't related. I changed it because I felt that a the route for login may not always be named that way and may be set in the configurations. Having the named of the route hardcoded may lead to 404 errors in future

Reply:
OK, so it shouldn't be changed in this PR.
Please revert, if you think we need this change, please send in a new PR thread.

Reply:
Noted, I'll revert then add that to another pull request.